### ISSUE TITLE
Write sync_cleared_for_campus_data.R

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#70]
Write a script called `sync_cleared_for_campus_data.R` that synchronizes `cleared_for_campus_timestamp` and `cleared_for_campus` between Result and Cases and Result and Contacts. This script assures that each project in each pair of projects has the latest data for the fields `cleared_for_campus_timestamp` and `cleared_for_campus`. 

The synchronization will be bidirectional between a pair of projects. The field `cleared_for_campus_timestamp` will indicate which project in the pair has the more recent data that needs to be copied to the other project. The fields copied will be `cleared_for_campus_timestamp` and `cleared_for_campus`. 

Implement the synchronization in a function named `sync_fields_by_timestamp`. the args to the function will be

```
dataset_a
join_field_in_a
dataset_b
join_field_in_b
timestamp_field
fields_to_sync
```

The returned value should be a list with the values 

```
a_writes
b_writes
```

Call this function twice: once for results/cases and once fro results/contacts.  Row bind the outputs that pertain to the results project and then write to each of the three input datasets.

**Also**
1. Add email logging to report on activity whenever nrow() for one of the datasets if non-zero. All such output should be summary only. It should say how many row were written to each project.
1. Add a crontab entry in the example file to run this script every 5 minutes.

**Comment by ChemiKyle** [comment]
@pbchase are you opposed to hardcoding `fields_to_sync` as `cleared_for_campus` and `cleared_for_campus_timestamp` in rev 1?

**Comment by pbchase** [comment]
@ChemiKyle, not only is that the rev 1 plan, that is the _only_ plan. Taryn renamed the fields in question to simplify your code. Do not bother making it all flexible and fancy.

**Comment by ChemiKyle** [comment]
Addressed by #76
